models/image_classification/model_templates/custom-classification/mobilenet_v3_small/compression_config.json

```diff
@@ -0,0 +1,46 @@
+{
+    "base": {
+        "nncf_config": {
+            "compression": [],
+            "log_dir": "."
+        }
+    },
+    "nncf_quantization": {
+	"nncf": {
+	    "enable": true
+	},
+	"train": {
+	    "batch_size": 64
```

Comment:
is `batch_size` always the same inside particular configuration file? Or can we agree on it (that is it the same)? i am worried about user who won't be able to modify batch size in case of OOM for example.

Reply:
Yes, it should be pointed in config file, but it will be really small, so I do not think that some issues may happen